modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/discovery/ShreddedType.scala

```diff
@@ -113,24 +122,22 @@ object ShreddedType {
 
   /** Regex to extract `SchemaKey` from `shredded/good` */
   val ShreddedSubpathPattern: Regex =
```

Comment:
Do we still need this now that loader doesn't need to list folders and can use directly the content of SQS message to know what to load ?

Reply:
Not much. And actually most of the module is not used, but as I mentioned previously with `listS3` I still think we'll find use for this to check batch consistency. But I'll create a ticket to delete this if it won't be useful.